workflows/files/ZWECONF.xml

```diff
@@ -480,19 +480,52 @@ How we want to verify SSL certificates of services. Valid values are:
             <default>false</default>
         </boolean>
     </variable>
-    <variable name="components_gateway_apiml_security_auth_provider" scope="instance" visibility="public">
-        <label>Gateway APIML security auth provider</label>
-        <abstract>Authorization provider for the gateway</abstract>
-        <description>Authorization provider for the gateway</description>
+    <variable name="components_zaas_enabled" scope="instance" visibility="public">
+      <label>Enable gateway</label>
```

Comment:
probably forgotten update of label, 
![image](https://github.com/user-attachments/assets/0b3a9263-7f24-4479-9e1b-51f0bf332ba5)